faq.html

```diff
@@ -139,17 +139,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
         <div class="pycon-faq">
           <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></h5>
           <ul class="faq-answer">
-              <li> - Workshop/Developers Sprints days (Aug 25 and Aug 26, 2018): 08:00 AM - 06:30 PM.</li>
-              <li>- Conference days (Aug 22 to Aug 24, 2018): 08:00 AM </li>
+              <li> - Workshop/Developers Sprints days (Oct 8 and Oct 9, 2018): 08:00 AM - 06:30 PM.</li>
+              <li>- Conference days (Oct 6 and Oct 7, 2018): 08:00 AM </li>
               <li>- 5.30 PM.</li>
           </ul>
         </div> 
         <div class="pycon-faq">
-          <h5><b>Q. What all does the Conference ticket (Aug 22 to Aug 24, 2018) include? </b></h5>
+          <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li> - Full conference pass for three days.</li>
```

Comment:
Conference ticket is for 2 days (Oct 6 - 7).
For Workshop and Dev Sprints there will be separate ticket. Can you update this as well ?

Reply:
@akshitac8 the emphasis here is on `three days` which needs to be changed to `two days`.